README.md

```diff
@@ -21,7 +21,9 @@ You need to have [.NET Core](https://dotnet.microsoft.com/download) installed an
 ```bash
 git clone https://github.com/PlayFab/MpsAgent.git
 cd LocalMultiplayerAgent 
-dotnet build
+# replace the LocalMultiplayerAgentPublishFolder with the folder of your choice
+dotnet publish --framework netcoreapp3.1 --runtime win-x64 -c Release -o LocalMultiplayerAgentPublishFolder -p:PublishSingleFile=true -p:PublishTrimmed=true --self-contained true

```

Comment:
any reason for the framework? that should be specified in the project file itself

Reply:
good catch, fixed